docs/how-to-docs/anchore-scan.md

```diff
@@ -86,6 +86,55 @@ steps:
     - push
     - tag
 ```
+#### Adding a CSV file within `WHITELIST`
+We could also add a centralised place to list all vulnerabilities that we would like to whitelist.
+
+We can achieve this by doing the following 4 steps
+
+
+1. You need to make a repo for this example will call it my-cve-exceptions
+
+2. Add a CSV In the root directory called `list.csv`
+
+```
+#This is the centralise list for all vulnerabilities that we would like to whitelist.
```

Comment:
CSV stands for 'comma-separated values', but this is a list with values on individual lines.
Might be better if the file had a `.txt` extension, but no biggie...